modules/core/src/core/model-utils.js

```diff
@@ -1,17 +1,85 @@
 import GL from '@luma.gl/constants';
 import {Buffer} from '@luma.gl/webgl';
+import {assert} from '../utils';
 
-export function getBuffersFromGeometry(gl, geometry) {
-  const attributes = geometry.getAttributes();
+// Support for mapping new geometries with glTF attribute names to "classic" luma.gl shader names
+const GLTF_TO_LUMA_ATTRIBUTE_MAP = {
```

Comment:
Not sure I like the coupling to glTF. As I mentioned above, I think it would streamline things to have loader.gl use the Luma naming conventions, if that's not too big a change. Otherwise, I'd prefer the name mapping be pulled out into a helper function like `getGeometryFromLoadersgl` or `getBuffersFromLoadersGL` so none of Luma's internals (including the geometry classes) need to know about it.

Reply:
Same as prev: update per discussion - we'll gradually replace the old luma.gl conventions with glTF naming.